ports/stm32/Makefile

```diff
@@ -342,6 +342,8 @@ SRC_O += \
 endif
 endif
 
+$(SRC_O): CFLAGS += -Wdouble-promotion -Wfloat-conversion
```

Comment:
Can these warnings be applied to the global CFLAGS?  Before this commit, with `MICROPY_FLOAT_IMPL=MICROPY_FLOAT_IMPL_FLOAT`, the `Wdouble-promotion` was actually part of the global CFLAGS but now it's not.  So ideally this would be globally applied.

Reply:
Same reason as above, but yes the `Wdouble-promotion` should stay where it was.